salt/orchestrate/services/scylladb.sls

```diff
@@ -0,0 +1,153 @@
+{% from "orchestrate/aws_env_macro.jinja" import VPC_NAME, VPC_RESOURCE_SUFFIX,
+ ENVIRONMENT, BUSINESS_UNIT, subnet_ids with context %}
+{% set INSTANCE_COUNT = salt.environ.get('INSTANCE_COUNT', 3) %}
+{% set scylla_admin_password = salt.random.get_str(42) %}
+
+load_scylladb_cloud_profile:
+  file.managed:
+    - name: /etc/salt/cloud.profiles.d/scylladb.conf
+    - source: salt://orchestrate/aws/cloud_profiles/scylladb.conf
+    - template: jinja
+
+generate_cloud_map_file:
+  file.managed:
+    - name: /etc/salt/cloud.maps.d/{{ VPC_RESOURCE_SUFFIX }}_scylladb_map.yml
+    - source: salt://orchestrate/aws/map_templates/instance_map.yml
+    - template: jinja
+    - makedirs: True
+    - context:
+        num_instances: {{ INSTANCE_COUNT }}
+        service_name: scylladb
+        tags:
+          business_unit: {{ BUSINESS_UNIT }}
+        environment_name: {{ ENVIRONMENT }}
+        roles:
+          - scylladb
+        securitygroupid:
+          - {{ salt.boto_secgroup.get_group_id(
+            'scylladb-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
+          - {{ salt.boto_secgroup.get_group_id(
+            'salt_master-{}'.format(ENVIRONMENT), vpc_name=VPC_NAME) }}
+        subnetids: {{ subnet_ids }}
+    - require:
+        - file: load_scylladb_cloud_profile
+
+deploy_scylladb_nodes:
+  salt.runner:
+    - name: cloud.map_run
+    - path: /etc/salt/cloud.maps.d/{{ VPC_RESOURCE_SUFFIX }}_scylladb_map.yml
+    - kwargs:
+        parallel: True
+    - require:
+        - file: generate_cloud_map_file
+
+format_data_drive:
+  salt.function:
+    - tgt: 'G@roles:scylladb and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - blockdev.formatted
+    - kwarg:
+        name: /dev/xvdb
+        fs_type: xfs
+    - require:
+        - salt: deploy_elasticsearch_nodes
+
+mount_data_drive:
+  salt.function:
+    - tgt: 'G@roles:scylladb and G@environment:{{ ENVIRONMENT }}'
+    - tgt_type: compound
+    - name: state.single
+    - arg:
+        - mount.mounted
+    - kwarg:
+        name: /var/lib/scylla
+        device: /dev/xvdb
```

Comment:
Isn't the drive /dev/xvda2?

Reply:
No, xvda2 would be partition 2 on the xvda block device.